mod_modular_vanilla/hooks/skills/actives/reap_skill.nut

```diff
@@ -0,0 +1,14 @@
+::ModularVanilla.MH.hook("scripts/skills/actives/reap_skill", function(q) {
+	// Convert the vanilla method of "setting" certain fields to instead be incremental changes
+	q.onAfterUpdate = @() function( _properties )
+	{
+		if (_properties.IsSpecializedInPolearms)
+		{
+			this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
+			if (this.m.ActionPointCost > 5)
```

Comment:
Like this, order matters, when multiple effects discount the cost of such a skill.
I suggest 
- you just reduce the AP cost by 1 no matter what. That is still in line with how the vanilla mastery actually works
- you make the discount dependant on what the BaseActionPointCost of the skill is. If it is 6+, then there is a discount.

Reply:
Will go with the approach of reducing by 1 if Base ActionPointCost is greater than 5.